src/Parser/BloodDeathKnight/Modules/Items/T20_4pc.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const RuinicDiscound=5;
+
+class T20_2pc extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  DeathAndDecayCounter = 0;
+  RunicPowerSaved=0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasBuff(SPELLS.BLOOD_T20_4PC_GRAVEWARDEN.id);
+    console.log('You have the 4pc');
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.DEATH_AND_DECAY.id) {
+      this.DeathAndDecayCounter++;
+    }
+  }
+
+
+  item() {
+    this.RunicPowerSaved=this.DeathAndDecayCounter*RuinicDiscound;
```

Comment:
This should either be a local variable (e.g. `const runicPowerSaved...`), or the runicPowerSaved should be a getter;

```jsx
get runicPowerSaved() {
  return this.deathAndDecayCasts * RuinicDiscound;
}
```

Reply:
fixed